components/ca-crud.html

```diff
@@ -704,7 +716,9 @@
             if (previewable) {
                 arr.splice(0, 0, 'Preview');
             }
-            const buttons = Object.keys(labelToAction).concat(arr);
+
+            // add the action and navigation buttons to the base buttons for the dialog
+            const buttons = Object.keys(labelToAction).concat(Object.keys(labelToNav)).concat(arr);
```

Comment:
So long as it was considered. On the TC project a non-uniqueness filter on the UI caused some issues (I think it was actually in ca-crud)